sitzverteilung-frontend/src/components/result/ResultTable.vue

```diff
@@ -116,28 +157,52 @@ function getResultColumns() {
           width: 50,
         },
       ],
-    };
-  });
-}
-
-function getValidationColumns() {
-  return AVAILABLE_METHODS.map((method) => {
-    return {
-      title: CALCULATION_METHOD_SHORT_FORMS[method],
-      key: `${method}${ResultDataSuffix.validationSuffix}`,
-      width: 60,
-    };
-  });
-}
+    })),
+  },
+  {
+    title: "Dokumentation",
+    width: 200,
+    key: "documentation",
+  },
+];
 
 const dialog = ref(false);
 const detailTitle = ref("");
-
-// Placeholder for future content, currently unused
 const detailInfo = ref<string>("");
 
 function goToDetail(selectedCalculationMethod: CalculationMethod) {
   detailTitle.value = selectedCalculationMethod;
   dialog.value = true;
 }
+
+function generateStaleText(item: ResultData, method: CalculationMethod) {
+  const staleInfo = props.unmappedResults.methods[method]?.stale;
+
+  if (staleInfo !== undefined) {
+    if (staleInfo.groupNames.includes(item.name)) {
+      return "Patt zwischen: " + staleInfo.groupNames.join(", ");
+    }
+  }
+}
+
+function generateValidationText(
+  item: ResultData,
+  method: CalculationMethod
+): string {
+  const validationData =
+    props.unmappedResults?.methods[method]?.validation?.[item.name];
+  if (!validationData) return "Keine Validierungsdaten vorhanden";
+
+  const reasons = [
+    ...(validationData.overRounding ? ["Überrundung"] : []),
+    ...(validationData.lostSafeSeat ? ["Verlust eines sicheren Sitzes"] : []),
+    ...((validationData.committeeInvalid?.length ?? 0) > 0
+      ? [
+          `Ungültige Ausschussgruppen: ${validationData.committeeInvalid.join(", ")}`,
+        ]
```

Comment:
Die Fehlermeldung `committeeInvalid` zeigt an, welche Partei einer AG zu stark ist um diese AG zu bilden. Ich würde daher eher einen Wortlaut wie

```suggestion
      ? [
          `Konstellation ungültig, wegen ${validationData.committeeInvalid.join(", ")}`,
        ]
```
bevorzugen.